package.json

```diff
@@ -6,6 +6,15 @@
   "repository": "https://github.app/EdgeApp/edge-currency-plugins",
   "license": "MIT",
   "author": "Airbitz, Inc.",
+  "exports": {
+    ".": {
+      "import": "./lib/index.mjs",
+      "require": "./lib/index.js"
+    }
+  },
+  "main": "./lib/index.js",
+  "module": "./lib/index.mjs",
```

Comment:
This is invalid. You cannot use the mjs file as the "module" entry point, since the mjs file is just a wrapper around the CJS file. The "module" entry is supposed to be a native ESM file, to allow bundlers like Webpack to do special optimizations not possible using CommonJS.